lib/_foreign_function.js

```diff
@@ -6,6 +6,7 @@
 var assert = require('assert')
   , debug = require('debug')('ffi:_ForeignFunction')
   , ref = require('ref')
+  , Promise = require('any-promise')
```

Comment:
Do we need `any-promise`? Node 0.10 is extremely old, and standard Promises were introduced in Node 0.12. Could we just deprecate 0.10? Node 6 just came out.